Orange/widgets/unsupervised/owdistancetransformation.py

```diff
@@ -50,8 +50,11 @@ def __init__(self):
                          btnLabels=[x[0] for x in self.inversion_options],
                          callback=self._invalidate)
 
-        gui.auto_commit(self.controlArea, self, "autocommit", "Apply",
-                        checkbox_label="Apply on any change")
+        box = gui.widgetBox(self.controlArea, True, orientation="vertical")
+        box = gui.auto_commit(self.controlArea, self, "autocommit", "Apply",
+                              checkbox_label="Apply on any change")
+        gui.separator(box, 20)
+        box.layout().addWidget(self.report_button)
```

Comment:
Let's keep the @wadhwasahil's chunk until @kernc adds smart Report button placement to `OWWidget`. :) Then I volunteer to review all widgets and remove any unnecessary code.

@wadhwasahil, it looks OK for merging, but we'll do it tomorrow. Most work - and in particular most merges - are done on Fridays, when the core team meets.

@kernc, I agree that `OWWidget` should do something reasonable, but it's not trivial: consider that Report button is constructed in `OWWidget.__init__`, so you can't put it into the Apply box that does not even exist yet. Even worse, `OWDataSampler` and some other widgets replace the Report button just to pull it from `OWWidget.leftArea` to `OWWidget.controlArea`, to make it a bit narrower. Hm, what about using the `WidgetMetaClass` to decorate `__init__` so that it calls a "post-init" function, which puts the report button at the bottom of `self.controlArea` unless it is already laid out?